pkg/config/config.go

```diff
@@ -49,3 +52,38 @@ func GetDefaultConfigPath() (string, error) {
 
 	return filepath.Join(home, DefaultConfigPath, DefaultConfigFileName), nil
 }
+
+func GetPathToCalculation(defaultPath string, fileName string) (string, error) {
+	user, err := user.Current()
+	if err != nil {
+		return "", fmt.Errorf("couldn't get user's information: %v", err)
+	}
+	home := user.HomeDir
+
+	if defaultPath == "" {
+		return filepath.Join(home, DefaultConfigPath, fileName), nil
+	} else {
+		if _, err := os.Stat(defaultPath); os.IsNotExist(err) {
+			logrus.Info("Desired path to download the calculations does not exist. The calculations are set to be downloaded into: $HOME/.config/ccboc/{id}")
+			return filepath.Join(home, "/.config/ccboc/", filepath.Base(fileName)), nil
+		}
+		defaultPath = filepath.Join(defaultPath, filepath.Base(fileName))
```

Comment:
There is no need to manipulate this variable that was passed in the function already.